pkg/startup/startup.go

```diff
@@ -1365,3 +1370,54 @@ func extractKubeadmCIDRs(kubeadmConfig *v1.ConfigMap) (string, string, error) {
 
 	return v4, v6, err
 }
+
+// Check if the string is cidr
+// Since: internal/bytealg/indexbyte_generic.go
+func indexByteString(s string, c byte) int {
+	for i := 0; i < len(s); i++ {
+		if s[i] == c {
+			return i
+		}
+	}
+	return -1
+}
+
+// return the cidr of the dest on local
+func getLocalCidr(ip string, version int) (string, error) {
+	i := indexByteString(ip, '/')
```

Comment:
Might be simpler to just check if the string has a `/` in it using the `strings` package:

```
if strings.Contains(ip, "/") {
  // Already a CIDR
  return ip, nil
}
```

Then we don't need to copy the `indexByteString` function here locally.

Reply:
fix